src/foam/core/types.js

```diff
@@ -129,7 +130,32 @@ foam.CLASS({
       function (_, v) {
         return typeof v === 'number' ? v : v ? parseFloat(v) : 0.0 ;
       }
-    ]
+    ],
+    [
+      'hashPropertyValue',
+      function(value) {
+        this.float64Array[0] = value;
+        return foam.util.hashTogether(
+            foam.util.hashCode(this.int32Array[0]),
+            foam.util.hashCode(this.int32Array[1]));
+      }
+    ],
+    [
+      'arrayBuffer',
+      new ArrayBuffer(8)
```

Comment:
[dup] Email replies don't stay threaded:

It was to avoid using a closure. The typed arrays need to pass in the
buffer that backs them. We could, instead, use a closure.